packages/as-transformer/src/transformers/massaExport.ts

```diff
@@ -319,13 +319,262 @@ export class MassaExport {
       );
     }
 
-    // checking if 'import { encode } from "as-base64";' is needed or if they are already imported by the contract
-    const encodeBase64ImportRegex =
-      /(?:import\s*{.*encode.*}\s*from\s*("|')as-base64("|'))/gm;
-
-    if (encodeBase64ImportRegex.exec(content) === null) {
-      imports.push('import { encode } from "as-base64";');
-    }
+    imports.push(`\n// adapted from https://gist.github.com/Juszczak/63e6d9e01decc850de03
```

Comment:
@Elli610 can you create an non urgent issue to explicitly link the source of this code ?

Reply:
of course